lib/src/otmorris/Morris.hxx

```diff
@@ -41,7 +41,7 @@ public:
   Morris();
 
   /** Standard constructor with in/out designs */
-  Morris(const OT::Sample & inputSample, const OT::Sample & outputSample);
+  Morris(const OT::Sample & inputSample, const OT::Sample & outputSample, const OT::Interval & interval);
 
```

Comment:
I think we could also get Morris(const OT::Sample & inputSample, const OT::Sample & outputSample);
In that case, we consider interval_ = Interval(ot.Point(inputSample.getDimension(), 0), ot.Point(inputSample.getDimension(), 1))

Reply:
Yes we could do that but I am afraid that the user will forget to put the bounds when it is really needed.

Reply:
Ok thus we should mention changes in Changelog

Reply:
Yes right with a new version